src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/PagingQueryString.feature.disabled

```diff
@@ -0,0 +1,202 @@
+Feature: Query String handling for GET requests for Resource Queries
+
+@ignore
```

Comment:
These two case sensitivity scenarios are good, but should move to the other file since they are not related to paging.

Reply:
Removed